### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by bertvannuffelen**
A selected number of updates resolving a number of editor notes.

Preview:

https://raw.githack.com/bertvannuffelen/GeoDCAT-AP/develop/drafts/latest/index.html

### CODE REVIEW

File: drafts/latest/index.html [hunk 1]
```diff
@@ -135,17 +135,18 @@ <h3>Scope of the revision</h3>
...
-<p>To be revised</p>
+<p>To be revised: adjust to official release of DCAT-AP. Plus propagate this through the text.</p>
 </aside>
 
 <!--
 <p>The DCAT-AP Application Profile defined in this document is based on the specification of the Data Catalog Vocabulary (DCAT) of 16 January 2014 [[VOCAB-DCAT-20140116]], and the Data Catalog Vocabulary (DCAT) - Version 2, W3C Candidate Recommendation 03 October 2019 [[VOCAB-DCAT-2-20191003]].</p>
 -->
-
-<p>The DCAT-AP Application Profile defined in this document is based on the DCAT-AP specification of 20 September 2019 [[DCAT-AP-20191120]], and it has been designed in conformance with the guidelines for the creation of DCAT-AP extensions [[DCAT-AP-EG]].</p>
+<p>The specification GeoDCAT-AP is designed as extension of DCAT-AP in conformance with the guidelines for the creation of DCAT-AP extensions [[DCAT-AP-EG]]. The DCAT-AP Application Profile on which this document is based is the DCAT-AP specification of 20 September 2019 [[DCAT-AP-20191120]].  According to the same principles DCAT-AP is in itself an extension of DCAT.
```
Comment (andrea-perego) [inline drafts/latest/index.html:147]:
```suggestion
<p>The specification GeoDCAT-AP is designed as extension of DCAT-AP in conformance with the guidelines for the creation of DCAT-AP extensions [[DCAT-AP-EG]]. The DCAT-AP Application Profile on which this document is based is the DCAT-AP specification of 20 September 2019 [[DCAT-AP-20191120]].  According to the same principles DCAT-AP is in itself an extension of DCAT.
```
```suggestion
<p>The specification GeoDCAT-AP is designed as an extension of DCAT-AP in conformance with the guidelines for the creation of DCAT-AP extensions [[DCAT-AP-EG]]. The DCAT-AP Application Profile on which this document is based is the DCAT-AP specification of 20 September 2019 [[DCAT-AP-20191120]].  According to the same principles DCAT-AP is in itself an extension of DCAT.</p>
```

File: drafts/latest/index.html [hunk 2]
```diff
@@ -135,17 +135,18 @@ <h3>Scope of the revision</h3>
...
+<p>
```
Comment (andrea-perego) [inline drafts/latest/index.html]:
```suggestion

```

File: drafts/latest/index.html [hunk 3]
```diff
@@ -135,17 +135,18 @@ <h3>Scope of the revision</h3>
...
+<p>This dependency must be taken into account while reading and using this specification. When implementing GeoDCAT-AP first these specifications should be consulted to fill in any missing gaps in this document.</p>
```
Comment (andrea-perego) [inline drafts/latest/index.html]:
```suggestion
<p>This dependency must be taken into account while reading and using this specification. When implementing GeoDCAT-AP, these specifications should be consulted first, to fill in any missing gaps in this document.</p> 
```

File: drafts/latest/index.html [hunk 4]
```diff
@@ -154,12 +155,15 @@ <h3>The DCAT-AP specification</h3>
 <section id="terminology">
 
 <h2>Terminology used in the Application Profile</h2>
+<aside class="ednote">
+<p>This section has been copied from DCAT-AP. It covers 2 aspects, a) (a partial set of) definitions of the domain specific core concepts and b) an reading guide on specification terminology.
```
Comment (andrea-perego) [inline drafts/latest/index.html]:
```suggestion
<p>This section has been copied from DCAT-AP. It covers 2 aspects, a) (a partial set of) definitions of the domain specific core concepts and b) an reading guide on specification terminology.</p>
```

File: drafts/latest/index.html [hunk 5]
```diff
@@ -154,12 +155,15 @@ <h3>The DCAT-AP specification</h3>
 <section id="terminology">
 
 <h2>Terminology used in the Application Profile</h2>
+<aside class="ednote">
+<p>This section has been copied from DCAT-AP. It covers 2 aspects, a) (a partial set of) definitions of the domain specific core concepts and b) an reading guide on specification terminology. 
+Proposal is to remove the definitions of the domain specific terminology and leave that to the appropriate places below. </p>
```
Comment (andrea-perego) [inline drafts/latest/index.html]:
```suggestion
<p>The proposal is to remove the definitions of the domain specific terminology and leave that to the appropriate places below.</p>
```

File: drafts/latest/tables/class-relationship.html [hunk 6]
```diff
@@ -17,6 +17,7 @@
 <th>Usage note</th>
 <td>
 <p>An association class for attaching additional information to a relationship between DCAT Resources</p>
+<p>To be used only when the nature of the relationship is known but is not adequately characterized by the existing properties in this specification.</p>
```
Comment (andrea-perego) [inline drafts/latest/tables/class-relationship.html]:
```suggestion
<p>To be used only when the nature of the relationship is known but is not adequately characterised by the existing properties in this specification.</p> 
```

File: drafts/latest/tables/optional-properties-for-dataset.html [hunk 7]
```diff
@@ -4,6 +4,7 @@
 <p>This property refers to information that indicates whether the Dataset is open data, has access restrictions or is not public. A controlled vocabulary with three members (<code>:public</code>, <code>:restricted</code>, <code>:non-public</code>) will be created and maintained by the Publications Office of the EU.</p>
 </blockquote>
 <p>A code list with those values have been actually published by the OP [[?EUV-AR]], so the usage note should be updated accordingly.</p>
+<p>An issue has been raised in DCAT-AP: <a href"https://github.com/SEMICeu/DCAT-AP/issues/159">issue 159</a></p>
```
Comment (andrea-perego) [inline drafts/latest/tables/optional-properties-for-dataset.html]:
```suggestion
<p>An issue has been raised in DCAT-AP: <a href="https://github.com/SEMICeu/DCAT-AP/issues/159">issue 159</a>.</p>
```

**Review by andrea-perego — CHANGES_REQUESTED** [review]
Thanks, @bertvannuffelen .

I added some inline comments proposing editorial changes.

**Comment by bertvannuffelen** [comment]
ok, I will include them.

kr,

Bert

Op di 29 sep. 2020 om 16:52 schreef Andrea Perego <notifications@github.com

**Comment by andrea-perego** [comment]
BTW, to accept the changes, you just have to go to

https://github.com/SEMICeu/GeoDCAT-AP/pull/17

The proposed revisions are all listed there, and there is a button to
commit the changes.

About the diagram, I noticed that not all the GeoDCAT-AP specific
properties are included (see, e.g., class Agent). Is this intentional?

Thanks

Andrea

File: drafts/latest/index.html (hunk 1 above)
Comment (andrea-perego) [inline drafts/latest/index.html]:
```suggestion
<p>The GeoDCAT-AP specification is designed as an extension of DCAT-AP in conformance with the guidelines for the creation of DCAT-AP extensions [[DCAT-AP-EG]]. The DCAT-AP Application Profile on which this document is based is the DCAT-AP specification of 20 September 2019 [[DCAT-AP-20191120]].  According to the same principles DCAT-AP is in itself an extension of DCAT.</p>
```

**Pull request merged**